tests/app.test.js

```diff
@@ -0,0 +1,102 @@
+const app = require("../routes/app.js");
+const supertest = require("supertest");
+const { router } = require("../routes/urlshortner.js");
+const request = supertest(router);
+const mainServerRequest = supertest(app);
+const fs = require("fs");
+const { describe, it } = require("@jest/globals");
+const testUrl =
+  "https://www.google.com/search?q=adadsa&oq=adadsa&aqs=chrome..69i57j0i10l9.708j0j7&sourceid=chrome&ie=UTF-8";
+const alreadyShortenedUrl =
+  "https://stackoverflow.com/questions/48248832/stylesheet-not-loaded-because-of-mime-type";
+let oldCount;
+let oldShortID;
+
+describe("URL SHORTENER API", () => {
+  it("should be able to get shortened URL", async (done) => {
+    let longUrl = testUrl;
+    requestBody = { longUrl };
+    const response = await request.put("/api/shorturl/").send(requestBody);
+    const { short } = JSON.parse(response.text);
+    let isShortID = false;
+    short.padStart("/");
+    if (short.match(/\b\/.*/g)) {
+      isShortID = true;
+    }
+    oldShortID = short;
+    expect(response.status).toBe(200);
+    expect(isShortID).toBe(!null);
+    done();
+  });
```

Comment:
You are testing `GET` but sending `PUT`?
To match regex patterns you can use [this](https://jestjs.io/docs/expect#expectstringmatchingstring--regexp)